src/main/java/seedu/eduke8/question/Question.java

```diff
@@ -44,6 +43,10 @@ public boolean wasShown() {
         return wasShown;
     }
 
+    public void markAsShown() {
```

Comment:
I see what you are trying to do here but is there a reason for this class to be different from the other displayables and do we need this extra method?

Reply:
Initially I wanted the getDescription method to only be getting the description of the question, but it is also indeed true that once getDescription method is called it is clear that the question is shown too. Will be changing this! Thanks!